rskj-core/src/main/java/org/ethereum/core/Transaction.java

```diff
@@ -471,6 +471,10 @@ public byte[] getEncoded() {
         return ByteUtil.cloneBytes(this.rlpEncoding);
     }
 
+    public long getSize() {
+        return this.getEncoded().length;
```

Comment:
as transactions sometimes may be quite big objects, do you think we could extract `getEncoded()` method into some private, let's say, `getEncodedInternal()`, which does not do `ByteUtil.cloneBytes(this.rlpEncoding)` inside, and use it here instead? - just to prevent useless copies of byte arrays

Reply:
Good point, I did not realise we were doing the clone 👍

Reply:
Done